### PR TITLE
Machines endpoints JSON API compliant

### PR DESCRIPTION
**Pull request by willrnch**
Machines endpoints now use the `jsonapi` lib for jsonapi serialisation and return standard errors. 
Add licence on plaza files.

### CODE REVIEW

File: nanocloud/errors/errors.go [hunk 1]
```diff
@@ -61,16 +61,39 @@ var (
...
-	WindowsNotOnline = &apiError{
+	InvalidMarchineStatus = &apiError{
 		0x000007,
-		http.StatusServiceUnavailable,
-		"Windows is not available.",
+		http.StatusBadRequest,
+		"The specified machine status is not valid.",
 	}
 
-	NeedFirstConnection = &apiError{
```
Comment (dynamiccast) [inline nanocloud/errors/errors.go:70]: Why was this error message removed ?

File: nanocloud/errors/errors.go [hunk 2]
```diff
@@ -61,16 +61,39 @@ var (
...
+	}
+
+	UnableToRetrieveMachineList = &apiError{
+		0x000010,
+		http.StatusInternalServerError,
+		"The machine list cannot be retrieved.",
+	}
+
+	WindowsNotOnline = &apiError{
+		0x000011,
```
Comment (dynamiccast) [inline nanocloud/errors/errors.go:95]: This error code is expected not to change since frontend is checking for it sometimes.

**Comment by dynamiccast** [comment]
No description provided

**Comment by dynamiccast** [comment]
There are still no tests for api/machine

File: nanocloud/errors/errors.go (hunk 2 above)
Comment (willrnch) [inline nanocloud/errors/errors.go:95]: i know

**Pull request merged**